src/main/java/net/snowflake/ingest/streaming/internal/FlushService.java

```diff
@@ -20,13 +13,7 @@
 import java.security.InvalidAlgorithmParameterException;
 import java.security.InvalidKeyException;
 import java.security.NoSuchAlgorithmException;
-import java.util.ArrayList;
-import java.util.Calendar;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.TimeZone;
+import java.util.*;
```

Comment:
Nit, please expand this. We avoid doing wildcard imports as per our style guide.

Reply:
ups, ide, done